.github/workflows/NativeTemplateDev6.yml

```diff
@@ -0,0 +1,34 @@
+name: '[Android] Custom Dev App CI'
+
+on:
+  schedule: 
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Actions need to be referenced by full commit sha, for security (you'll also see one of the checks failed on this)